README.md

```diff
@@ -1,9 +1,16 @@
-# Shoulda Matchers [![Gem Version][version-badge]][rubygems] [![Build Status][travis-badge]][travis] ![Downloads][downloads-badge] [![Hound][hound-badge]][hound]
+# Shoulda Matchers
+[![Gem Version][version-badge]][rubygems]
```

Comment:
This places all of the badges after the header, is that intentional? Fine if so, but just checking.